components/sup/src/manager/mod.rs

```diff
@@ -592,30 +616,24 @@ impl Manager {
                 .write()
                 .expect("Services lock is poisoned!")
                 .iter_mut() {
-            if self.updater
-                   .check_for_updated_package(service, &self.census_ring) {
-                service.populate(&self.census_ring);
-                self.gossip_latest_service_rumor(&service);
```

Comment:
hmm. did you find this gossiping was not necessary? I'd think it would be something you would want to rumor out but maybe something else takes care of the rumor. Just curious about the rationale.